Makefile.rules

```diff
@@ -1268,6 +1271,27 @@ testlibgap: ${LIBGAPTESTS}
 		$(v) -A -l $(top_srcdir) -q -T --nointeract >$(v).out && \
 		diff $(top_srcdir)/$(v).expect $(v).out && ) :
 
+# test libgap's pkg-config's libgap version reporting
+PKG_REPORTED_VERSION := $(shell PKG_CONFIG_PATH=$(DESTDIR)$(libdir)/pkgconfig pkg-config --modversion libgap)
+
+tst/testpkgconfigversion: install-libgap
+ifeq ($(PKG_REPORTED_VERSION), $(PACKAGE_VERSION))
+	@echo "pkg-config reports correct version of libgap: "$(PKG_REPORTED_VERSION)
+else
+	@echo "pkg-config does not report version of libgap, or is not installed"
+endif
+
+# test libgap's pkg-config's libgap flags fetching
+PKG_REPORTED_CFLAGS := $(shell PKG_CONFIG_PATH=$(DESTDIR)$(libdir)/pkgconfig pkg-config --cflags libgap | cut -d' ' -f 1)/gap
+PKG_REPORTED_LDFLAGS := $(shell PKG_CONFIG_PATH=$(DESTDIR)$(libdir)/pkgconfig pkg-config --libs libgap)
+v := "tst/testlibgap/basic"
+tst/testpkgconfigbuild: install build/obj/tst/testlibgap/common.c.lo
+	$(QUIET_CC)$(CC) -c $(v).c -o $(v).o $(PKG_REPORTED_CFLAGS)
+	$(QUIET_LINK)$(LINK) $(v).o -o $(v) build/obj/tst/testlibgap/common.c.lo $(PKG_REPORTED_LDFLAGS)
+	$(v) -A -l $(top_srcdir) -q -T --nointeract >$(v).out && \
+        diff $(top_srcdir)/$(v).expect $(v).out
```

Comment:
Doing this kind of testing in the Makefile is quite tortured.

Since the test seems to expect GAP to be installed, I would suggest to add it into `dev/ci.sh`, into the `testmakeinstall` target there.

Reply:
How about keeping it in the makefile, and just calling `make` on the corresponding targets in  `dev/ci.sh` ?

Reply:
> How about keeping it in the makefile, and just calling `make` on the corresponding targets in `dev/ci.sh` ?

done in https://github.com/gap-system/gap/pull/5080/commits/46f9c2388ad8881d0fb3e873ac68d769ad778567